packages/astro/src/vite-plugin-astro/hmr.ts

```diff
@@ -108,12 +101,6 @@ export async function handleHotUpdate(
 		}
 	}
 
-	// TODO: Svelte files should be marked as `isSelfAccepting` but they don't appear to be
-	const isSelfAccepting = mods.every((m) => m.isSelfAccepting || m.url.endsWith('.svelte'));
-	if (!isSelfAccepting) {
-		logger.debug('watch', 'full page reload triggered');
-	}
```

Comment:
Not sure if we need this special log check. Besides the Svelte handling, all modules that are self-accepting doesn't necessarily trigger a full page reload 🤔

Reply:
I added this a long time ago to workaround a Svelte bug at the time. I'm sure it's out of date and no longer needed.